btcjson/walletsvrcmds.go

```diff
@@ -677,6 +682,205 @@ func NewWalletPassphraseChangeCmd(oldPassphrase, newPassphrase string) *WalletPa
 	}
 }
 
+// Timestamp defines a type to represent a timestamp value in seconds,
+// since epoch.
+//
+// The value can either be a integer, or a string (ex, "now").
+//
+// NOTE: Interpretation of the timestamp value depends upon the specific
+// JSON-RPC command, where it is used.
+type Timestamp struct {
```

Comment:
Should it be called `TimestampOrNow` as this can either be an integer timestamp or the string "now"?

Other option would be the more generic `TimestampOrString` but I think it would be a good idea to specify that the string value must be "now" as required in the spec.

Reply:
Totally agree. Renamed to `TimestampOrNow` and added a small check to make sure the string is always `"now"`.

Reply:
Awesome the check is a good idea 👌